docs/tutorial/force_calibration.rst

```diff
@@ -1,380 +1,440 @@
+The various theoretical models that can be used to fit these data are described in the :doc:`theory section on force calibration</theory/force_calibration/force_calibration>`, while their use is described below.
 
-    P(f) = \frac{D_\mathrm{measured}}{\pi ^ 2 \left(f^2 + f_c ^ 2\right)} g(f, f_\mathrm{diode}, \alpha) \quad \mathrm{[V^2/Hz]}
+.. image:: figures/force_calibration/diode_filtering.png
+  :nbattach:
 
-where :math:`D_\mathrm{measured}` corresponds to the diffusion constant (in `volts`), :math:`f` the frequency and :math:`f_c` the corner frequency.
-The first part of the equation is known as the Lorentzian model and represents the part of the spectrum that originates from the physical motion of the bead.
-The second term :math:`g` takes into account the slower response of the position detection system and is given by:
-
-.. math::
-
-    g(f, f_\mathrm{diode}, \alpha) = \alpha^2 + \frac{1 - \alpha ^ 2}{1 + (f / f_\mathrm{diode})^2}
+.. warning::
 
-Here :math:`\alpha` corresponds to the fraction of the signal response that is instantaneous, while :math:`f_\mathrm{diode}` characterizes the frequency response of the diode. Note that not all sensor types require this second term.
+    For high corner frequencies, the parameter estimation procedure can become unreliable.
+    A warning sign for this is when the corner frequency `f_c` approaches or goes beyond the diode frequency `f_diode`.
+    For more information see the section on `High corner frequencies`_.
 
-To convert the parameters obtained from this spectral fit to a trap stiffness, the following is computed:
+Undoing the previous calibration
+--------------------------------
 
-.. math::
+Our starting point will be data acquired with Bluelake, which is already calibrated.
+Therefore, the first step is to undo the calibration applied by Bluelake to get the raw voltage signals.
+Let's load the dataset::
 
-    \kappa = 2 \pi \gamma_0 f_c \quad \mathrm{[N/m]}
+    f = lk.File("passive_calibration.h5")
 
-Here :math:`\kappa` then represents the final trap stiffness.
+The force timeline in this file contains a section used for calibration.
+We can see calibrations relevant to this file by inspecting the `calibration` attribute for the entire force :class:`~lumicks.pylake.channel.Slice`.
+This is a list of dictionaries containing all of the relevant calibration parameters::
 
-We can calibrate the position by considering the diffusion of the bead:
+    print(len(f.force1x.calibration))
+    print(f.force1x.calibration[1])
 
-.. math::
+The first calibration always refers to the calibration that was active at the start of the slice (usually from a calibration that occurred before the start of the slice).
+Our goal here is to recalibrate the data in the calibration section (corresponding to `calibrations[1]`).
+First we grab the chunk of data that was used for that calibration in Bluelake::
```

Comment:
yeah I still find this confusing. I think it could help to use the terms "calibration measurement" and "calibration applied to ....", part of the confusion here I think is just re-used terminology so being more explicit in this case I think is helpful.

But what really made it clear to me was plotting things out. could add a code block to make a figure like this:

![image](https://user-images.githubusercontent.com/61475504/222148375-f081161d-f2a5-4fc2-9314-ede722fff472.png)

the code I used is as follows (but I dunno if it's still too complicated to put in the tutorial, but I don't see a significantly better way to clean it up)

```
force = f.force1x

# final timestamp in data relative to first timestamp, in seconds
stop = (force.timestamps[-1] - force.timestamps[0]) * 1e-9

# measurement timestamp ranges for each calibration relative to first timestamp, in seconds
calibrations = [
    (np.array([cal["Start time (ns)"], cal["Stop time (ns)"]]) - force.timestamps[0]) * 1e-9
    for cal in force.calibration
]

plt.figure()
force.plot()
plt.axvspan(*calibrations[0], alpha=0.5, fc="tab:blue", label="calibration[0] measurement")
plt.axvspan(calibrations[0][1], calibrations[1][0], alpha=0.2, fc="tab:blue", label="calibration[0] applied")
plt.axvspan(*calibrations[1], alpha=0.5, fc="tab:orange", label="calibration[1] measurement")
plt.axvspan(calibrations[1][1], stop, alpha=0.2, fc="tab:orange", label="calibration[0] applied")

plt.legend()
plt.margins(x=0)
# plt.xlim(0, stop)  # uncomment this line to zoom in to data
    
plt.tight_layout()
plt.show()
```

Reply:
While I like the idea of having this in the docs, I fear it'll distract from the force calibration part of the docs, because people might get hung up on the complexity.

Instead I made the following image, which I propose we use instead (along with some further minor improvements on the wording, differentiating calibration from measurement).

![calibrations](https://user-images.githubusercontent.com/19836026/222209334-c74e657a-c9f8-4e4c-8eaf-32b952ddceb6.png)